README.md

```diff
@@ -1,22 +1,22 @@
-# generator-wasm-oci [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url]
+# generator-wasm [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url]
 
 Generate WASM modules that can be pushed to OCI registries, for example
 for use with [Krustlet](https://github.com/deislabs/krustlet).
 
 **This generator is currently work in progress.  You cannot yet install it using npm.**
 If you would like to try it, you can clone
-the repo and run `npm run compile && npm link` to hook it up to Yeoman so that you can run `yo wasm-oci`.
+the repo and run `npm run compile && npm link` to hook it up to Yeoman so that you can run `yo wasm`.
 Or let us know so that we're motivated to get it ready for a proper release!
 
 ## Installation
 
 First, install [Yeoman](http://yeoman.io) using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
 
-**NOTE: You can't install `generator-wasm-oci` using `npm install` yet - for now you need to run `npm run compile && npm link` as described above.**  ~~Then install `generator-wasm-oci` also using `npm`.~~
+**NOTE: You can't install `generator-wasm` using `npm install` yet - for now you need to run `npm run compile && npm link` as described above.**  ~~Then install `generator-wasm` also using `npm`.~~
```

Comment:
You have access to publish to NPM if you want to.

Reply:
Yeah, will update this as part of a "publish 0.0.1" changeset - thanks!